Add github issue template (#320)
deltachat/deltachat-desktop: .github/ISSUE_TEMPLATE.md

<!--

> Please fill in as much as you can below (leaving out if not applicable is ok). This is a bug report forum, if you like to discuss a new feature or submit suggestions, please use the community forum at https://support.delta.chat

Reviewer: ```suggestion
Please fill in as much as you can below (leaving out if not applicable is ok). This is a bug report and feature-tracking forum, if you like to discuss a new feature, please use the community forum at https://support.delta.chat. 
```
